bundles/core/src/main/java/com/adobe/cq/commerce/core/search/internal/services/SearchResultsServiceImpl.java

```diff
@@ -44,6 +47,7 @@
 import com.adobe.cq.commerce.core.search.services.SearchFilterService;
 import com.adobe.cq.commerce.core.search.services.SearchResultsService;
 import com.adobe.cq.commerce.graphql.client.GraphqlResponse;
+import com.adobe.cq.commerce.magento.graphql.*;
```

Comment:
Sorry I insist, we don't do `*` imports. ;-)

Reply:
Argh, I fixed that but my IDE replaced it. I fixed it this time